cyclus_input_gen/templates.py

```diff
@@ -574,16 +572,24 @@ class template_collections:
 """
 
     candu_template_cyborg = """
-    <prototypes>
-	{{ prototype }}
-</prototypes>
-<build_times>
-	{{ start_time }}
-</build_times>
-<n_build>
-	{{ number }}
-</n_build>
-<lifetimes>
-	{{ lifetime }}
-</lifetimes>
-"""
\ No newline at end of file
+    <facility>
+    <!-- {{ country }} -->
+    <!-- {{ type }}    -->
+    <name>{{reactor_name}}</name>
+    <config>
+      <Cyborg>
+        <power_cap>{{capacity}}</power_cap>
```

Comment:
are we using cyborg?

Reply:
nope, not in this work but I long ago when I wanted to try out Cyborg I made it and it was running (at that time of cyborg)... Should I delete these?